### PR TITLE
docs: NFT Documentation Update #3

### PR DESCRIPTION
**Pull request by astarinmymind**
- adds receipt format
- adds more information to factories

todo: 
- add provider.getNFT

### CODE REVIEW

**Comment by keriat** [comment]
@astarinmymind is this one still relevant?

**Comment by keriat** [comment]
@bxpana @astarinmymind Mates 👋 Shell we..? I mean let's either close it or merge it

File: docs/dev/nfts.md [hunk 1]
```diff
@@ -139,6 +139,19 @@ To get a receipt for the minted NFT:
 const receipt = await nft.awaitReceipt();
 ```
 
+Receipt format is as follows:
+
+```typescript
+pub struct TxReceiptResponse {
```
Comment (StanislavBreadless) [inline docs/dev/nfts.md:145]: This is Rust struct, but the tutorial is for JS.

Why do we need to add it here? I guess it is better suited for the general API

**Comment by StanislavBreadless** [comment]
Oh, it is from May

**Comment by StanislavBreadless** [comment]
@astarinmymind please answer the questions and re-open if needed

**Pull request closed without merging**